docs/HOWTO.md

```diff
@@ -277,6 +281,62 @@ Then the datasets can be declared in the benchmark definition file as follow:
+
+For time series forecasting tasks, the dataset must be stored as a single [CSV] file in [long format](https://doc.dataiku.com/dss/latest/time-series/data-formatting.html#long-format).
+
+The CSV file must include 3 columns:
+  - unique ID of each time series
+  - timestamp of the observation
+  - target value of the time series
+
+
+_Example:_
+
+The following file contains 2 time series with daily frequency: `A` with 3 observations and `B` with 4 observations.
+
+| IdColumn |	TimestampColumn |	TargetColumn |
+|---------|-----------|--------|
+| A       |	2020-01-01|	2.0    |
+| A       |	2020-01-02|	1.0    |
+| A       |	2020-01-03|	5.0    |
+| B       |	2019-01-01|	8.0    |
+| B       |	2019-01-01|	2.0    |
+| B       |	2019-01-01|	1.0    |
+| B       |	2019-01-01|	9.0    |
+
+Suppose the above table is saved as a CSV file under `/path/to/data.csv`.
+
+The respective task definition may look like:
+
+```yaml
+- name: example_time_series_data
+  dataset:
+    path: /path/to/data.csv
+    freq: D
+    forecast_horizon_in_steps: 1
+    seasonality: 7
+    id_column: IdColumn  # defaults to 'item_id'
+    timestamp_column: TimestampColumn  # defaults to 'timestamp'
+    target: TargetColumn
+  folds: 1
+```
+The configuration of a time series task **must** include the following parameters:
+- `path` - a local or remote (e.g., S3) path to the CSV file with time series data.
+- `freq` - a [pandas-compatible frequency string](https://pandas.pydata.org/docs/user_guide/timeseries.html#offset-aliases) that denotes the frequency of the time series. For example, `D` for daily, `H` for hourly, or `15min` for 15-minute frequency.
+- `forecast_horizon_in_steps` - a positive integer denoting how many future time series values need to be predicted.
+- `seasonality` - a positive integer denoting the seasonal period of the data that is used for computing metrics like [mean absolute scaled error](https://en.wikipedia.org/wiki/Mean_absolute_scaled_error#Seasonal_time_series).
+
+
+**Note**:
+- AMLB automatically generates the train & test splits from the raw data depending on the chosen `forecast_horizon_in_steps` and `fold` parameters. For each time series `x` in the datasets, the folds are constructed as:
+  - In fold `0`, use `x[:-forecast_horizon_in_steps]` as the train set and `x[-forecast_horizon_in_steps:]` as the test set (forecast horizon).
+  - In fold `1`, use `x[:-2*forecast_horizon_in_steps]` as the train set and `x[-2*forecast_horizon_in_steps:-forecast_horizon_in_steps]` as the test set.
+  - In fold `n`, use `x[:-(n+1)*forecast_horizon_in_steps]` as the train set and `x[-(n+1)*forecast_horizon_in_steps:-n*forecast_horizon_in_steps]` as the test set.
+- Time series may have different lengths or have different starting timestamps, but need to have the same frequency
+- The shortest time series in the dataset must have length of at least `folds * forecast_horizon_in_step + 1`
+- All time series must have regular timestamp index
+
```

Comment:
Adding the Table from our discussion would reduce the strain on the reader and make it clear at a glance.